docs/docs/sbom-tools.md

```diff
@@ -0,0 +1,34 @@
+For SBOM file generation, some SBOM tools are required to be installed in your environment:
```

Comment:
For Software Bill of Materials (SBOM) file generation, some SBOM tools are required to be installed in your environment:

Reply:
OK